docs/modules/ROOT/pages/index.adoc

```diff
@@ -288,21 +288,24 @@ FeatureHub was designed to ensure that security was built into the platform from
 === Key Security concepts
 
 ==== Administrators
-There are two types of administrators, *Site Administrators* and *Portfolio Administrators*.
+There are two types of administrators, *Organisation Super Admin* and *Portfolio Admin*.
 
 ===== Site Administrators
```

Comment:
You changed the name to "Organisation Super Admin" but didn't change the title here. 🍷